src/AccessibilityInsights.Rules/EvaluationCode.cs

```diff
@@ -8,12 +8,42 @@ namespace Axe.Windows.Rules
     /// </summary>
     public enum EvaluationCode
     {
-        NotApplicable, // given element did not meet the conditions for the rule
+        /// <summary>
+        /// The given element did not meet the conditions for the rule and was not evaluated.
+        /// </summary>
+        NotApplicable,
+
+        /// <summary>
+        /// The given element met the success criteria  of the rule evaluation.
```

Comment:
There is an extra whitespace character between 'criteria' and 'of the rule evaluation'